azerothcore/azerothcore-wotlk
# feat(Core/Config): Add Legacy Arena Points config option

## Changes Proposed:
-  Chooses if you want to use pre wotlk point calculation for season 6 or less

## Issues Addressed:


## SOURCE:


## Tests Performed:

- tested


## How to Test the Changes:


1. set Arena.LegacyArenaPoints = 1 or Arena.LegacyArenaPoints = 0 and Arena.ArenaSeason.ID = <6
2. (probably want to lower  Arena.GamesRequired = 10 to Arena.GamesRequired = 1) and Arena.AutoDistributePoints = 0 to 1
3. complete a game 2v2
4. close worldserver, set date 1 week ahead, let it auto distrubute points
5. notice that you gain very few points with legacyarenapoints 1 and way more points when 0

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
